src/components/ReportActionItem/IOUQuote.js

```diff
@@ -17,6 +17,16 @@ const propTypes = {
     /** Callback invoked when View Details is pressed */
     onViewDetailsPressed: PropTypes.func,
 
+    /** Current logged in User's name */
+    currentUserName: PropTypes.string.isRequired,
```

Comment:
Is it correct for these props to be required?

Reply:
I am thinking we would always need to show the name of the user somewhere in the transaction. Can remove `isRequired` if you think that won't be the case.

Reply:
Yeah, I think these are safe. The only one that may not be is `sessionEmail`. It can be null when passed in. I think to fix it, just make `sessionEmail` default to `''` instead of `null` on line `62` of `IOUAction`

Reply:
That way we don't end up passing `null` to a `string.isRequired` prop, because that could lead to us unsafely doing something like `sessionEmail.replace(...)` and causing a NPE.

Reply:
Noted. Thanks for the feedback.

Reply:
Updated